### PR TITLE
GetPerson service - added another method

### PR DESCRIPTION
**Pull request by chiaramapellimt**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PersonControllerTest.kt [hunk 1]
```diff
@@ -300,24 +310,28 @@ internal class PersonControllerTest(
...
+              },
+            "pncId": null,
+            "hmppsId": null,
+            "contactDetails": null
           },
-          "pncId": null,
-          "hmppsId": null,
-          "contactDetails": null
-        }
+          "prisonerOffenderSearch": null
```
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PersonControllerTest.kt:331]: Can we have a test to verify prisonerOffenderSearch data is populated please?

Comment (chiaramapellimt) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/controllers/v1/person/PersonControllerTest.kt:331]: okay!

**Review by CslBing — APPROVED** [review]
Looks good

**Comment by CslBing** [comment]
Update open doc yml please

**Pull request merged**